zeppelin-server/src/main/java/org/apache/zeppelin/service/ShiroAuthenticationService.java

```diff
@@ -415,26 +423,39 @@ private List<String> getUserList(JdbcRealm obj) {
     try {
       dataSource = (DataSource) FieldUtils.readField(obj, "dataSource", true);
       authQuery = (String) FieldUtils.readField(obj, "authenticationQuery", true);
-      LOGGER.info(authQuery);
+      LOGGER.debug("authenticationQuery={}", authQuery);
       String authQueryLowerCase = authQuery.toLowerCase();
       retval = authQueryLowerCase.split("from", 2);
       if (retval.length >= 2) {
         retval = retval[1].split("with|where", 2);
-        tablename = retval[0];
+        tablename = retval[0].trim();
```

Comment:
Just a little thing.
Meanwhile, I would use `strip()` because this method is Unicode aware. What do you think?

```suggestion
        tablename = retval[0].strip();
```